tests/unittests/test_backoff.py

```diff
@@ -808,3 +812,401 @@ def test_no_socket_timeout_error_occurred__subscriber(self, mocked_write_records
         # verify if 'tap_exacttarget.dao.DataAccessObject.write_records_with_transform' was called
         # once as there is only one record
         self.assertEquals(mocked_write_records.call_count, 1)
+
+@mock.patch("time.sleep")
+class TestTimeoutError(unittest.TestCase):
```

Comment:
@harshpatel4crest  Can we add a testcase when URLError occurs with string that **does not contain** timed out?

Reply:
Added test case.